ngmix/tests/test_flags.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+import pytest
+from ..flags import get_flags_str, NAME_MAP, LOW_DET
+
+
+@pytest.mark.parametrize(
+    "dtype", [np.int8, np.uint8, np.int16, np.uint16, np.int32, np.uint32, np.int64, np.uint64]
+)
+def test_get_flags_str(dtype):
+    """Test that get_flags_str works for different integer types."""
+    val = np.array(LOW_DET, dtype=dtype)
```

Comment:
For the bigger types we should or with 2**45 to ensure things are OK.

Reply:
In `numpy<2`, bitwise or to `uint64` is prohibited for some reason. I'll condition on that if that's okay.

Reply:
Ack I forgot and yes pls do!